### DISCUSSION TITLE
如果上行下行以及读写的字体大小能修改就好了

### DISCUSSION THREAD

**Discussion by wuyilala** [lollipopkit/flutter_server_box#240] [General]
目前看上去，上行下行速度的字体太小了，看起来比较麻烦。如果可以自定义字体大小就方便了。类似Servercat一样的字体大小。

**Comment by lollipopkit** [comment]
已在最新版本中实现

**Reply by wuyilala** [reply]
太棒了，下次要是可以修改字体颜色就好了。
servercat的服务器界面配色很舒适，最好可以提供。
